walk-in-interview/src/main/java/com/google/job/data/Requirement.java

```diff
@@ -26,6 +27,22 @@ public String getRequirementId() {
         return requirementId;
     }
 
+    /** Gets the requirement stable ids given enum value. */
+    public static List<String> getRequirementIds(List<Requirement> requirements) {
```

Comment:
can this be private?

Reply:
also, why is this method necessary, isn't getRequirementId() enough

Reply:
getRequirementId() is for single Requirement, getRequirementIds() is for a specific list of requirements